etna/transforms/feature_selection/feature_importance.py

```diff
@@ -235,8 +235,14 @@ def _fit(self, df: pd.DataFrame) -> "MRMRFeatureSelectionTransform":
         df_features = df_features.loc[df_features.first_valid_index() :]
         relevance_table = self.relevance_table(df_target, df_features, **self.relevance_params)
 
+        if relevance_table.values.min() < 0:
```

Comment:
you should add it into relevance_table, not here
Here you should have guarantees that all values >= 0

Reply:
Why? We can use relevance table for some other things, and only in MRMR we need non negative relevance

Reply:
From any relevance table the importance must be non-negative just because it is relevance